aws/resource_aws_dax_cluster_test.go

```diff
@@ -269,11 +269,23 @@ func testAccCheckAWSDAXClusterExists(n string, v *dax.Cluster) resource.TestChec
 	}
 }
 
-var baseConfig = `
-provider "aws" {
-  region = "us-west-2"
+func testAccPreCheckAWSDax(t *testing.T) {
+	conn := testAccProvider.Meta().(*AWSClient).daxconn
+
+	input := &dax.DescribeClustersInput{}
+
+	_, err := conn.DescribeClusters(input)
+
+	if testAccPreCheckSkipError(err) || isAWSErr(err, "InvalidParameterValueException", "Access Denied to API Version: DAX_V3") {
+		t.Skipf("skipping acceptance testing: %s", err)
+	}
+
+	if err != nil {
+		t.Fatalf("unexpected PreCheck error: %s", err)
+	}
 }
 
+var baseConfig = `
```

Comment:
I would recommend a const as opposed to a var here. 
```suggestion
const baseConfig = `
```

Reply:
Ah yeah, I didn't fix that when removing the provider config, I'll update. 👍